test/test_integration_check.py

```diff
@@ -37,18 +36,26 @@ def setUp(self):
     # *************** TESTS ***************
 
     @integration_test()
-    def test_check_privilege_insecure_prints_warning_in_log(self):
+    def test_check_default_role_get_insecure_prints_warning_in_log(self):
```

Comment:
created 15 major tests and should all pass.

i.e.:
test_check_default_role_get_insecure_prints_warning_in_log (test.test_integration_check.CliIntegrationTestCheck) ... ok

test_check_existent_privilege_on_non_permitted_resource_with_non_permitted_specified_role (test.test_integration_check.CliIntegrationTestCheck) ... ok
...